tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/StoragesDsl.scala

```diff
@@ -0,0 +1,91 @@
+package ch.epfl.bluebrain.nexus.tests.kg.files
+
+import akka.http.scaladsl.model._
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.ClasspathResourceLoader
+import ch.epfl.bluebrain.nexus.testkit.CirceEq
+import ch.epfl.bluebrain.nexus.tests.Identity.storages.Coyote
+import ch.epfl.bluebrain.nexus.tests.Optics.{filterKey, filterMetadataKeys}
+import ch.epfl.bluebrain.nexus.tests.{CirceUnmarshalling, HttpClient}
+import io.circe.Json
+import org.scalatest._
+import org.scalatest.matchers.should.Matchers
+
+import scala.sys.process._
+
+class StoragesDsl(deltaClient: HttpClient) extends CirceUnmarshalling with Matchers with CirceEq {
+
+  private val loader = ClasspathResourceLoader()
+
+  def createStorage(payload: Json, projectRef: String): IO[Assertion] =
+    deltaClient.post[Json](s"/storages/$projectRef", payload, Coyote) { (_, response) =>
+      response.status shouldEqual StatusCodes.Created
+    }
+
+  def createDiskStorageDefaultPerms(id: String, projectRef: String): IO[Assertion] =
```

Comment:
I think this needs to say `createDiskStorageWithDefaultPerms` because it sounds like you're creating the default permissions

Reply:
(Same with the other methods)